src/Qdrant.Client/Grpc/Query.cs

```diff
@@ -0,0 +1,111 @@
+namespace Qdrant.Client.Grpc;
+
+/// <summary>
+/// A query to perform.
+/// </summary>
+public partial class Query
+{
+	/// <summary>
+	/// Implicitly converts a <see cref="RecommendInput"/>  to a new instance of <see cref="Query"/>
+	/// </summary>
+	/// <param name="input">An instance of <see cref="RecommendInput"/> </param>
+	/// <returns>a new instance of <see cref="Query"/></returns>
+	public static implicit operator Query(RecommendInput input) => new() { Recommend = input };
+
+	/// <summary>
+	/// Implicitly converts a <see cref="DiscoverInput"/>  to a new instance of <see cref="Query"/>
+	/// </summary>
+	/// <param name="input">An instance of <see cref="DiscoverInput"/> </param>
+	/// <returns>a new instance of <see cref="Query"/></returns>
+	public static implicit operator Query(DiscoverInput input) => new() { Discover = input };
+
+	/// <summary>
+	/// Implicitly converts a <see cref="ContextInput"/>  to a new instance of <see cref="Query"/>
+	/// </summary>
+	/// <param name="input">An instance of <see cref="ContextInput"/> </param>
+	/// <returns>a new instance of <see cref="Query"/></returns>
+	public static implicit operator Query(ContextInput input) => new() { Context = input };
+
+	/// <summary>
+	/// Implicitly converts a <see cref="Fusion"/>  to a new instance of <see cref="Query"/> to combine pre-fetch results.
+	/// </summary>
+	/// <param name="input">A <see cref="Fusion"/> value</param>
+	/// <returns>a new instance of <see cref="Query"/></returns>
+	public static implicit operator Query(Fusion input) => new() { Fusion = input };
+
+	/// <summary>
+	/// Implicitly converts a <see cref="OrderBy"/>  to a new instance of <see cref="Query"/>
+	/// </summary>
+	/// <param name="orderBy">An instance of <see cref="OrderBy"/> </param>
+	/// <returns>a new instance of <see cref="Query"/></returns>
+	public static implicit operator Query(OrderBy orderBy) => new() { OrderBy = orderBy };
+
+	/// <summary>
+	/// Explicitly creates a <see cref="Query"/> to order points by a payload field.
+	/// </summary>
+	/// <param name="key">Name of the payload field to order by</param>
+	/// <returns>a new instance of <see cref="Query"/></returns>
+	public static explicit operator Query(string key) => new() { OrderBy = new() { Key = key } };
```

Comment:
I've made this an explicit conversion because with

```csharp
await client.QueryAsync("{collection_name}", query: "bar");
```
Users might be misled into thinking that we're performing a semantic search with `"bar"`.

It should be
```csharp
await client.QueryAsync("{collection_name}", query: (OrderBy) "bar");
```